src/Request/Request.php

```diff
@@ -42,6 +42,11 @@ class Request implements ConfigAwareInterface, ContainerAwareInterface, LoggerAw
     const DEBUG_REQUEST_STRING = "#### REQUEST ####\nHeaders: {headers}\nURI: {uri}\nMethod: {method}\nBody: {body}";
     const DEBUG_RESPONSE_STRING =  "#### RESPONSE ####\nHeaders: {headers}\nData: {data}\nStatus Code: {status_code}";
 
+    /**
+     * @var array Names of the values to strip from debug output
+     */
+    protected $sensitive_data = ['machine_token', 'Authorization', 'session',];
```

Comment:
This permits classes extending `Request` to strip other sensitive data from the debug output.